src/test/java/bio/terra/service/filedata/azure/util/BlobIOTestUtility.java

```diff
@@ -157,6 +160,17 @@ public String uploadFileWithContents(String blobName, String contents) {
     }
   }
 
+  public String uploadControlFile(String blobName, List<BulkLoadFileModel> bulkLoadFileModelList)
+      throws JsonProcessingException {
+    ObjectMapper mapper = new ObjectMapper();
+    StringBuilder sb = new StringBuilder();
+    for (BulkLoadFileModel file : bulkLoadFileModelList) {
+      sb.append(mapper.writeValueAsString(file));
+      sb.append('\n');
+    }
+    return uploadFileWithContents(blobName, sb.toString());
```

Comment:
Since `blobName` is only used to call `uploadFileWithContents()`, another way to do this would be to write:
```
  public String readControlFile(List<BulkLoadFileModel> bulkLoadFileModelList)
      throws JsonProcessingException {
    ObjectMapper mapper = new ObjectMapper();
    StringBuilder sb = new StringBuilder();
    for (BulkLoadFileModel file : bulkLoadFileModelList) {
      sb.append(mapper.writeValueAsString(file));
      sb.append('\n');
    }
    return sb.toString();
  }
```
And then in the caller you'd write:
```
    String path = uploadFileWithContents(blobName, readControlFile(fileModels));
```

Reply:
Nice- thanks! With that change, does readControlFile() still belong in the BlobIOTestUtility?

Reply:
Probably not, it's not blob specific. Maybe put it in `TestUtils`? There's already an `objectMapper` there too.

Reply:
Good idea - I'll move there.